docs/index.rst

```diff
@@ -1,58 +1,22 @@
 
-Welcome to Rikai's documentation!
-=================================
+Welcome to Rikai!
+=================
 
-Rikai is an `Apache Spark`_ based ML data format built for working with
-unstructured data at scale. Processing large amounts of data for ML is never trivial, but that
-is especially true for images and videos often at the core of deep learning applications. We are
-building Rikai with two main goals:
+Rikai is a large-scale data system specicially designed for Machine Learning workflows,
```

Comment:
do we not want to call it a data format anymore? Or it's a data system with rikai format as a component?

Reply:
I like that the description is not all about Spark anymore.

Reply:
so this is format + SQL engine + visual ? feel that this is not just format anymore.